assets/k-image-with-copy-styles.css

```diff
@@ -16,9 +16,7 @@
   text-align: center;
 }
 .img-with-copy-para {
-  font-size: 24px;
```

Comment:
👍

Reply:
@heythisisnate this " font-size: 24px;" doesn't exist but anyway this will resolve once I dawnified all the custom sections